### DISCUSSION TITLE
Location of the AllocationRepository -> Fetching Module

### DISCUSSION THREAD

**Discussion by WLefever-Cegeka** [Informatievlaanderen/VSDS-LDESServer4J#830] [Ideas]
I'm wondering about the location of the AllocationRepository.
Within the Fragmentation Module the AllocationRepository is never queried by fragmentation itself, only updated.
So it might make more sense to move it to the Fetching Module, since there it's relevant.
In Fetching it can be updated using MemberAllocatedEvents and MemberUnallocatedEvents.

**Comment by Tomvbe** [comment]
Agreed in dev session to do this.
We will implement this in the future jira story to separate the fetch module.
